### PR TITLE
Stripped UserContext of membership data and helper functions

### PR DESCRIPTION
**Pull request by me-andre**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/domain/community/user/userContributions/useUserContributions.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    skip: !user?.user.id,
+  });
+
+  return useMemo(() => {
+    if (!data) {
+      return undefined;
+    }
+
+    const spaces = data?.rolesUser.spaces.map<ContributionItem>(e => ({
+      spaceId: e.nameID,
```
Comment (techsmyth) [inline src/domain/community/user/userContributions/useUserContributions.ts:22]: Should this be ID or nameID? Either the value or the name I think needs updating?

File: src/domain/community/user/userContributions/useUserContributions.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      e.challenges.map(c => ({
+        spaceId: e.nameID,
+        challengeId: c.nameID,
+        id: c.id,
+      }))
+    );
+
+    const opportunities = data?.rolesUser.spaces.flatMap<ContributionItem>(e =>
+      e.opportunities.map(o => ({
+        spaceId: e.nameID,
```
Comment (techsmyth) [inline src/domain/community/user/userContributions/useUserContributions.ts:36]: Should this be ID or nameID?

File: src/domain/community/user/providers/UserProvider/UserProvider.tsx [hunk 3]
```diff
@@ -37,23 +33,16 @@ const UserContext = React.createContext<UserContextValue>({
 const UserProvider: FC<{}> = ({ children }) => {
   const { isAuthenticated, loading: loadingAuthentication, verified } = useAuthenticationContext();
   // TODO "me" query fetches too much beyond user name
```
Comment (ccanos) [inline src/domain/community/user/providers/UserProvider/UserProvider.tsx:35]: I guess this TODO can be removed

File: src/domain/community/user/providers/UserProvider/UserProvider.tsx [hunk 4]
```diff
@@ -27,7 +24,6 @@ export interface UserContextValue {
 
 const UserContext = React.createContext<UserContextValue>({
   user: undefined,
-  userSpaceRoles: undefined,
   loading: true,
   loadingMe: true,
```
Comment (ccanos) [inline src/domain/community/user/providers/UserProvider/UserProvider.tsx:28]: still `loadingMe` ?

**Review by ccanos — DISMISSED** [review]
just a few small comments. I have approved because they are not even functional

**Review by ccanos — APPROVED** [review]

**Pull request merged**